packages/balance/config/paths.js

```diff
@@ -0,0 +1,58 @@
+const path = require('path')
+const fs = require('fs')
+const url = require('url')
+
+// Make sure any symlinks in the project folder are resolved:
+// https://github.com/facebook/create-react-app/issues/637
+const appDirectory = fs.realpathSync(process.cwd())
+const resolveApp = relativePath => path.resolve(appDirectory, relativePath)
+
+const envPublicUrl = process.env.PUBLIC_URL
+
+function ensureSlash(inputPath, needsSlash) {
+  const hasSlash = inputPath.endsWith('/')
+  if (hasSlash && !needsSlash) {
+    return inputPath.substr(0, inputPath.length - 1)
+  } else if (!hasSlash && needsSlash) {
+    return `${inputPath}/`
+  } else {
+    return inputPath
+  }
```

Comment:
```suggestion
  const hasSlash = inputPath.endsWith('/')
  if (hasSlash && !needsSlash) {
    return inputPath.substr(0, inputPath.length - 1)
  }
  if (!hasSlash && needsSlash) {
    return `${inputPath}/`
  }
  return inputPath
```

acho que como cada condicional tem um return poderia estruturar assim para  facilitar a leitura, o que acha?